pkg/apis/pipeline/v1beta1/taskrun_types.go

```diff
@@ -90,6 +93,14 @@ const (
 	TaskRunSpecStatusCancelled = "TaskRunCancelled"
 )
 
+// TaskRunSpecStatusMessage defines human readable status messages for the TaskRun.
+type TaskRunSpecStatusMessage string
+
+const (
+	// TaskRun cancelled as the Pipeline running it was cancelled.
```

Comment:
Please give this a valid godoc comment - i.e., one starting with `// TaskRunCancelledByPipelineMsg ...`

Reply:
Done